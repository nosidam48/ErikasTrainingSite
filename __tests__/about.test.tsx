import React from 'react';
import renderer from 'react-test-renderer';
import About from '../pages/about/index';

jest.mock('next/link', () => ({
  __esModule: true,
  default: ({ children, href }) => (
    <children.type {...children.props} href={href} />
  ),
}));

jest.mock('next/image', () => ({
  __esModule: true,
  default: (props: any) => {
    return <img {...props} />;
  },
}));

it('renders about unchanged', () => {
  const tree = renderer.create(<About />).toJSON();
  expect(tree).toMatchSnapshot();
});
