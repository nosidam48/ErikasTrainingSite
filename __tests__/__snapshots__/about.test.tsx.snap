// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders about unchanged 1`] = `
<div
  className="bioContainer"
>
  <div
    className="bio"
  >
    <div
      className="story"
    >
      <h1>
        Erika's Story
      </h1>
      <p>
        Erika is a lifelong dog lover with a passion for helping owners and their dogs develop better relationships through positive reinforcement training. She graduated from Northwest School of Canine Studies in 2015 and is a valued member of the teams at Seattle Puppyworks and Paws 4 Training. 
        <br />
        Working with Erika, you can expect to be met with relentless positivity and practical solutions to common behavioral issues. Erika is a puppy expert with years of experience working in puppy socialization studios. She can help with potty training, recall, and anything else you and your puppy may need. 
        <br />
        Erika also has extensive experience working with adult dogs. Her one-on-one approach and individualized training plans address your dog's specific needs and give you the tools to modify unwanted behaviors. Erika has worked with hundreds of clients all over Seattle and continues to learn and grow as a trainer every day.
      </p>
    </div>
    <div
      className="bioImage"
    >
      <img
        alt="Erika with her dog Garbanzo"
        height="426"
        priority={true}
        src="/images/erika-bonzo.jpg"
        width="426"
      />
    </div>
  </div>
  <div
    className="qualifications"
  >
    <div
      className="qualList"
    >
      <div
        className="imageWrapper"
      >
        <img
          alt=""
          height="91"
          src="/images/NWDog.png"
          width="100"
        />
      </div>
      <p>
        North West School of Canine Studies Graduate
      </p>
    </div>
    <div
      className="qualList"
    >
      <div
        className="imageWrapper"
      >
        <img
          alt=""
          height="72"
          src="/images/puppyworks.png"
          width="180"
        />
      </div>
      <p>
        Trainer and Instructor at Seattle Puppyworks
      </p>
    </div>
    <div
      className="qualList"
    >
      <div
        className="imageWrapper"
      >
        <img
          alt=""
          height="35"
          src="/images/paws4training.png"
          width="200"
        />
      </div>
      <p>
        Trainer and Instructor at Paws 4 Training
      </p>
    </div>
  </div>
</div>
`;
